Dockerfile

```diff
@@ -4,7 +4,8 @@ ENV DEBIAN_FRONTEND noninteractive
 
 # Get node and yarn
 RUN sudo apt-get update -y
-RUN apt-get install -y build-essential software-properties-common curl git && add-apt-repository ppa:ubuntu-toolchain-r/test
+RUN apt-get install -y build-essential software-properties-common curl git mercurial && add-apt-repository ppa:ubuntu-toolchain-r/test
+RUN apt-get update -y
```

Comment:
This second `apt-get update -y` seems redundant. Are you sure its needed for some reason?

Reply:
Yeah after adding the PPA.

Reply:
Oh, missed seeing the `add-apt-repository` in the previous line, never mind, this is fine :+1: